docs/ruleset_functions/expected_system_type_from_Table_G3_1_1.md

```diff
@@ -0,0 +1,117 @@
+# expected_system_type_from_Table_G3_1_1
+**Schema Version:** 0.0.22  
+
+**Description:** returns the expected system type based on the BAT, the number of floors of the building and the area of the BAT, and climate zone
+
+**Inputs:**
+- **building_area_type** - ("RESIDENTIAL", "PUBLIC_ASSEMBLY", "RETAIL", "HOSPITAL", "OTHER_NON_RESIDENTIAL", "HEATED-ONLY_STORAGE")
+- **number_of_floors** - this is the number of floors in the building integer greater than 0
+- **area_of_BAT** - this is the total area of the building area type
+
+**Returns:**  
+- **result**: a dict indicating the expected system type from table G3_1_1 ("SYS-1", "SYS-2", etc) and a string that indicates how the system was chosen, for example: {"EXPECTED_SYSTEM_TYPE": "SYS-4", "SYSTEM_ORIGIN": "PUBLIC_ASSEMBLY CZ_0_to_3a < 120,000 ft2"} - does not take into account G3.1.1 b-g
+ 
+**Function Call:**
+- **is_CZ_0_to_3a**
+
+## Logic:
+- create area_of_BAT_string, which will be appended to the details_of_system_selection string: `area_of_BAT_string = ""`
```

Comment:
would it be possible to save this as a number instead of a string? doing checks using strings can get messy and erroneous.

Reply:
`area_of_BAT_string` is integrated into the target string at the end of the function, and for residential buildings, there is no area component of the target string, so `area_of_BAT_string` stays equal to ""

the whole target string concept is a little messy - I'm open to other ideas on how to achieve the same goal.

Reply:
I think we should use the total gross conditioned floor area and number of floors in the entire building, not just that  BAT.  This is how G3.1.1 (a) is worded, and it was further clarified in 90.1 2022 per our email exchange with you and Christina.